management-service/services/experiment_service.go

```diff
@@ -578,6 +588,35 @@ func (svc *experimentService) filterExperimentStatusFriendly(query *gorm.DB, sta
 	return query.Where(orPredicates)
 }
 
+func (svc *experimentService) filterStartEndTimeValues(query *gorm.DB, params ListExperimentsParams) (*gorm.DB, error) {
```

Comment:
Created a helper method out of this because the linter was throwing out an excessive cyclomatic complexity for the `ListExperiments` method.